public/src/components/trips/updateTrip.jsx

```diff
@@ -1,120 +0,0 @@
-import React from 'react'
-import { connect } from 'react-redux';
-import {UpdateUserTrips} from '../../actions/myTripsActions.js';
-import getLocations from '../../actions/getLocationActions';
-import viewTrip from '../../actions/tripDetailsActions.js';
-import UpdateTripRow from './updateTripRow.jsx';
-
-class UserTripsRow extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      trip_id: 0,
-      tripdate: '',
-      time: '',
-      location_from: '',
-      location_to: '',
-      passing_by: '',
-      pass_point_time: '',
-      available_seats: 0
-    }
-  }
-  test(){
-    this.state = {
-      trip_id: this.props.GetTripByID.trip_id,
-      tripdate: this.props.GetTripByID.tripdate,
-      time: this.props.GetTripByID.time,
-      location_from: this.props.GetTripByID.location_from_id,
-      location_to: this.props.GetTripByID.location_to_id,
-      passing_by:this.props.GetTripByID.passing_by,
-      pass_point_time:this.props.GetTripByID.pass_point_time,
-      available_seats: this.props.GetTripByID.available_seats
-    };
-  }
-  componentWillMount(){
-    this.props.Locations();
-    {this.props.viewThisTrip(this.props.params.id)}
-  }
-
-  changeTripDate(ev) {
```

Comment:
same comment as above. make a reusable function to update different fields and parts of state.